services/brig/test/integration/API/Internal.hs

```diff
@@ -161,3 +81,81 @@ testEJPDRequest mgr brig brigep gundeck = do
             ]
     have <- ejpdRequestClient brigep mgr (Just True) req
     liftIO $ assertEqual "" want have
+
+testFeatureConferenceCallingByAccount :: forall m. TestConstraints m => Opt.Opts -> Manager -> Cass.ClientState -> Brig -> Endpoint -> Galley -> m ()
+testFeatureConferenceCallingByAccount (Opt.optSettings -> settings) mgr db brig brigep galley = do
+  let check :: HasCallStack => ApiFt.TeamFeatureStatusNoConfig -> m ()
+      check status = do
+        uid <- userId <$> createUser "joe" brig
+        _ <-
+          aFewTimes 12 (putAccountFeatureConfigClient brigep mgr uid status) isRight
+            >>= either (liftIO . throwIO . ErrorCall . ("putAccountFeatureConfigClient: " <>) . show) pure
+
+        mbStatus' <- getAccountFeatureConfigClient brigep mgr uid
+        liftIO $ assertEqual "1" (Right status) mbStatus'
```

Comment:
Could you use write some meaningful error messages here, so when these fail we know what went wrong?

Reply:
https://github.com/wireapp/wire-server/pull/1818/commits/2eecb795edfc4017a5e539dab8928fd8d6e4b0e9